### PR TITLE
Fix default refresh issue.

### PR DESCRIPTION
**Pull request by rmrichesjr1**
This should add a boolean attribute with a default value of true.

### CODE REVIEW

File: internal/provider/resource_apply_module.go [hunk 1]
```diff
@@ -172,6 +174,13 @@ func (t *applyModuleResource) Schema(_ context.Context, _ resource.SchemaRequest
 					stringplanmodifier.UseStateForUnknown(),
 				},
 			},
+			"refresh": schema.BoolAttribute{
+				MarkdownDescription: "Whether to do a Terraform refresh",
+				Description:         "Whether to do a Terraform refresh",
+				Optional:            true,
+				Computed:            true,
```
Comment (bbdev442) [inline internal/provider/resource_apply_module.go:181]: `computed` should not be specified here

File: docs/resources/apply_module.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ Defines and manages tharsis_apply_module resources, which launch runs in other w
 ### Optional
 
 - `module_version` (String) The version identifier of the module.
+- `refresh` (Boolean) Whether to do a Terraform refresh
```
Comment (bbdev442) [inline docs/resources/apply_module.md:26]: This description can be updated to describe refresh in more detail

**Comment by rmrichesjr1** [comment]
Removing the 'computed' option from the refresh attribute breaks acceptance tests and generate:
image

**Review by amanshallow — APPROVED** [review]

**Review by bbdev442 — APPROVED** [review]

**Comment by rmrichesjr1** [comment]
Need to test this with no refresh option, with refresh=true, and with refresh=false.

**Pull request merged**